dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/AggregatedFTPolicy.java

```diff
@@ -86,22 +78,16 @@ public Method getMethod() {
     }
 
     /**
-     * @return
+     * @return true if asyncResultWrapper is not null
```

Comment:
Inconsistent javadoc ... in most cases, you've removed it rather than filling it in properly. This javadoc just describes the implementation rather than telling you what the result means.

Reply:
Yep, I agree!